src/core.c/Order.pm6

```diff
@@ -28,12 +28,16 @@ multi sub infix:<cmp>(\a, Real:D \b) {
 }
 
 multi sub infix:<cmp>(Real:D \a, Real:D \b) {
-       (nqp::istype(a, Rational) && nqp::isfalse(a.denominator))
-    || (nqp::istype(b, Rational) && nqp::isfalse(b.denominator))
-    ?? a.Bridge cmp b.Bridge
-    !! a === -Inf || b === Inf
-        ?? Less
-        !! a === Inf || b === -Inf
+    nqp::istype(a,Rational) && nqp::istype(b,Rational)
```

Comment:
I wish this had some comments for every branch, otherwise it's very hard to deduce which cases are being tested for. The type graph for Real is rather complicated so maybe the comments themselves could reveal some bugs.